kytos/core/api_server.py

```diff
@@ -198,7 +198,9 @@ def get_ui_components(self, section_name):
 
     def web_ui(self):
         """Serve the index.html page for the admin-ui."""
-        return send_file(f"{self.flask_dir}/index.html")
+        if os.path.exists(f"{self.flask_dir}/index.html"):
+            return send_file(f"{self.flask_dir}/index.html")
+        return f"File '{self.flask_dir}/index.html' not found."
```

Comment:
Suggestion: use variable.

Question: shouldn't we return an HTTP error code somewhere?

Reply:
Yes